public/js/ChatController.js

```diff
@@ -60,7 +60,9 @@
         function refreshConversations(firstLoad) {
             RequestService.getConversations()
                 .then(function (result) {
-                    $scope.unseenMessages = ConversationService.updateCurrentConversations($scope.currentConversations, result.data);
+                    var unseen = ConversationService.updateCurrentConversations($scope.currentConversations, result.data);
+                    $scope.unseenMessages = unseen.unseenMessages;
```

Comment:
The idea was that the updateCurrentConversations function returns an object 'unseen' with two variables.
The two variables are then assigned to scope variables in the next two lines. Is there a better way of doing this?